examples/deadline/Local-Zone/python/package/lib/network_tier.py

```diff
@@ -0,0 +1,124 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import builtins
+import typing
+
+from aws_cdk.aws_ec2 import (
+    GatewayVpcEndpointAwsService,
+    InterfaceVpcEndpointAwsService,
+    SubnetConfiguration,
+    SubnetSelection,
+    SubnetType,
+    Vpc
+)
+from aws_cdk.aws_route53 import (
+    PrivateHostedZone
+)
+from aws_cdk.core import (
+    Construct,
+    Stack
+)
+import jsii
+
+from .config import config
+
+_INTERFACE_ENDPOINT_SERVICES = [
+    {'name': 'CLOUDWATCH', 'service': InterfaceVpcEndpointAwsService.CLOUDWATCH},
+    {'name': 'CLOUDWATCH_EVENTS', 'service': InterfaceVpcEndpointAwsService.CLOUDWATCH_EVENTS},
+    {'name': 'CLOUDWATCH_LOGS', 'service': InterfaceVpcEndpointAwsService.CLOUDWATCH_LOGS},
+    {'name': 'EC2', 'service': InterfaceVpcEndpointAwsService.EC2},
+    {'name': 'ECR', 'service': InterfaceVpcEndpointAwsService.ECR},
+    {'name': 'ECS', 'service': InterfaceVpcEndpointAwsService.ECS},
+    {'name': 'KMS', 'service': InterfaceVpcEndpointAwsService.KMS},
+    {'name': 'SECRETS_MANAGER', 'service': InterfaceVpcEndpointAwsService.SECRETS_MANAGER},
+    {'name': 'SNS', 'service': InterfaceVpcEndpointAwsService.SNS},
+    {'name': 'STS', 'service': InterfaceVpcEndpointAwsService.STS}
+]
+
+_GATEWAY_ENDPOINT_SERVICES = [
+    {'name': 'S3', 'service': GatewayVpcEndpointAwsService.S3},
+    {'name': 'DYNAMODB', 'service': GatewayVpcEndpointAwsService.DYNAMODB}
+]
+
+
+class NetworkTier(Stack):
+    """
+    The network tier consists of all constructs that are required for the foundational
+    networking between the various components of the Deadline render farm.
+    """
+
+    @builtins.property # type: ignore
+    @jsii.member(jsii_name="availabilityZones")
+    def availability_zones(self) -> typing.List[builtins.str]:
+        """
+        This overrides the availability zones the Stack will use. The zones that we set here are what
+        our VPC will use, so adding local zones to this return value will enable us to then deploy
+        infrastructure to them.
+        """
+        return config.availability_zones_standard + config.availability_zones_local
+
+    def __init__(self, scope: Construct, stack_id: str, **kwargs) -> None:
+        """
+        Initializes a new instance of NetworkTier
+        """
+        super().__init__(scope, stack_id, **kwargs)
+
+        # We're creating a SubnetSelection with only the standard availability zones
+        # to be used to put the NAT gateway in and the VPC endpoints, because the local zones do no have
+        # these available.
+        subnets = SubnetSelection(
+            availability_zones=config.availability_zones_standard,
+            subnet_type=SubnetType.PUBLIC
+        )
+
+        # The VPC that all components of the render farm will be created in.
+        # This overrides the availability zones the Stack will use. The zones that we set here are what
```

Comment:
Looks like this is a copy of the comment in `def availability_zones` above, but doesn't make sense in this context. Can we update this to say that we're using the override above for the `Vpc`?

Reply:
Oh ya that doesn't make sense. Updated.